### ISSUE TITLE
Quick Pick doesn't show borders that separate group

### ISSUE THREAD

**Issue by octref** [arcticicestudio/nord-visual-studio-code#100]
Nord:

![image](https://user-images.githubusercontent.com/4033249/45850556-74571280-bceb-11e8-93cb-d30aa8c4d24b.png)

Dark+:

![image](https://user-images.githubusercontent.com/4033249/45850574-7de07a80-bceb-11e8-9087-f9db90f1fb07.png)

**Comment by arcticicestudio** [comment]
Thanks for your report :+1: 

I've removed it in the first place because it made the whole drop-down look divided and broken into multiple pieces and a bit cluttered instead of smooth transitions into the next section. It was still distinguishable through the names of the labels on the right, so maybe changing the color of these to e.g. `nord8` might be better?

![gh-100-colored-label](https://user-images.githubusercontent.com/7836623/46306766-75265900-c5b5-11e8-98c5-07cd83df538c.png)

**Comment by octref** [comment]
Changing the group name is a welcoming change, but I'm not sure if that solves the original problem — it's hard to see which item on the left is on the same level as `Explorer`, so I cannot immediately determine what group `Outline` is in.

> Ah, and grats for joining the VS Code team, saw some of your implementations in latest versions (screenshots/screencasts with Nord theme 😄 ). Also using your nice polacode extension for ✨ code screenshots.

😉

**Comment by arcticicestudio** [comment]
Yeah, might be a bit uncomfortable to see this when you don't have a ruler at hand next to your monitor.
So I'll merge #103 and create a separate PR for the labels too. Thanks for your feedback.
